kommunicateui/src/main/res/values/strings.xml

```diff
@@ -115,6 +113,8 @@
     <string name="please_wait_creating_group_of_two">Please wait...</string>
     <string name="sort_contacts_by_lastSeenAt">Sort by lastSeen</string>
     <string name="sort_contacts_by_recent_chats">Sort by recent chats</string>
+    <string name="logout_info_text">"Logging out, please wait..."</string>
```

Comment:
@reytum some places we have .. and some places it is ... 
is there any specific convention on this?

Reply:
We usually use 3 dots for loading texts. 2 dots might be old code.

Reply:
@shubhamtewari saw you were using 2 dots in a recent commit, can you check whats the standard way based on Google guidelines?